tests/modules/module2.py

```diff
@@ -0,0 +1,36 @@
+from sqlalchemy import Column, UniqueConstraint, orm, types
+
+Base = orm.declarative_base()
+
+
+class TestUserMultilineDocstringD212(Base):
+    """
+    A ``user``.
+
+    The user has a multi-line docstring.
```

Comment:
```suggestion
    The user has a multi-line docstring::
    
       this is an indented code block
```

I think your current approach would fail with something like this

Reply:
Added test case